chore: release kong mesh 1.8.0 and 1.7.1 (#3989)
Kong/docs.konghq.com: app/mesh/changelog.md

no_version: true
---

## 1.8.0

> Released on **TBD**

Built on top of [Kuma 1.7.0](https://github.com/kumahq/kuma/releases/tag/1.7.0)

### Changes

New Features:

- Support for arm64
- Graceful shutdown of OPA

> - Role-based AWS auth for Vault
> - Add Vault AWS Auth option to set Server ID

Reviewer: ```suggestion
- Role-based AWS authentication for Vault
- Added a Vault AWS authentication option to set the server ID
```
What is the option? Can you add it as: "Added the `<option-name>` Vault AWS authentication option to set the server ID."

Author: I believe the option is called `Server ID` :-)

Author: > Added a Vault AWS authentication option to set the server ID header

Do you think it's ok now?

Dependency upgrades:

- Bump github.com/aws/aws-sdk-go from 1.40.56 to 1.44.21
- Bump github.com/hashicorp/go-retryablehttp from 0.6.6 to 0.7.1
- Bump github.com/open-policy-agent/opa from 0.38.1 to 0.40.0
- Bump github.com/open-policy-agent/opa-envoy-plugin from 0.38.1-envoy-3 to 0.40.0-envoy
- Bump k8s.io/api from 0.23.6 to 0.24.1
- Bump k8s.io/apimachinery from 0.23.6 to 0.24.1
- Bump sigs.k8s.io/controller-runtime from 0.11.2 to 0.12.1

### Upgrading

> ### kubectl

Reviewer: ```suggestion
#### kubectl
```
